spec/unit/metadata_spec.rb

```diff
@@ -80,7 +80,9 @@ module Omnibus
       it_behaves_like 'a version manipulator', 'fedora', '11.5', '11'
       it_behaves_like 'a version manipulator', 'freebsd', '10.0', '10'
       it_behaves_like 'a version manipulator', 'gentoo', '2004.3', '2004.3'
+      it_behaves_like 'a version manipulator', 'ios_xr', '6.0.0.14I', '6'
       it_behaves_like 'a version manipulator', 'mac_os_x', '10.9.1', '10.9'
+      it_behaves_like 'a version manipulator', 'nexus', '5.0', '5'
```

Comment:
@glennmatthews it doesn't really matter as this is just a unit test ensuring that whatever version is passed in is stripped down to its major version number for both "nexus" and "ios_xr" platforms.  I could have passed in 10.0.1.192123 as a version number for this unit under test, and provided it still returned 10, the test is operating properly.

With regards to the versions themselves, Ohai will be overwriting the VERSION field from /etc/os-release with /etc/cisco-release on WRL7, so the version from /etc/cisco-release will take precedence, which is why I used 6.0.0.14I in my test above.  On the WRL5 builder image I was provided, the VERSION field in /etc/os-release was simply "5" so I used "5.0" in my test above to ensure the string was truncated down to 5 as we expect.